client/app/services/routes.js

```diff
@@ -1,19 +1,40 @@
-import { isString, isObject, filter } from "lodash";
+import { isString, isObject, filter, sortBy, some } from "lodash";
+import pathToRegexp from "path-to-regexp";
+
+function isSimplePath(path) {
+  const tokens = pathToRegexp.parse(path);
+  return some(tokens, isObject);
+}
```

Comment:
This seems to be with the opposite logic

e.g.:
`/` is showing as not a simple path and `/alerts/:alertId` is showing as a simple path

Reply:
@gabrieldutra https://github.com/getredash/redash/pull/5020/commits/e45068a0ac0b289767b3ccf3aaba3eaf75c8c58e